### ISSUE TITLE
Code working with 1.13.2, not with 1.13.3

### ISSUE THREAD

**Issue by ws-way** [TomWright/dasel#102]
Hello @TomWright,

The code below is working with 1.13.2, but with 1.13.3 I do get an error:

```
Error: could not load input: could not unmarshal data: yaml: mapping values are not allowed in this context
```
```
echo '' | \
                dasel put string -p yaml [0].'job_name' 'logging-graylog' | \
                dasel put string -p yaml [0].'scrape_interval' '15s' | \
                dasel put string -p yaml [0].'basic_auth.username' admin | \
                dasel put string -p yaml [0].'basic_auth.password' 'password' | \
                dasel put string -p yaml [0].'metrics_path' '/api/plugins/org.graylog.plugins.metrics.prometheus/metrics' | \
                dasel put string -p yaml [0].'static_configs.[0].targets.[0]' 'logging.local'
```

Do you know why this is failing with 1.13.3 ?

Thanks!

**Comment by TomWright** [comment]
Hey @ws-way,

Thanks for raising this.

This was introduced as a result of the fix for #99. I've fixed this issue and added specific test cases for both issues.

This is fixed in `v1.13.4` 👍

**Comment by ws-way** [comment]
It's working now, thanks for the fast update
